integration_test/go_java_launcher_integration_test.go

```diff
@@ -69,6 +69,28 @@ func TestMainMethodContainerSupportEnabled(t *testing.T) {
 			launcherCustom:  "testdata/launcher-custom-initial-and-max-ram-percentage-override.yml",
 			expectedJVMArgs: "-XX\\:InitialRAMPercentage=79.9 -XX\\:MaxRAMPercentage=80.9 -XX\\:ActiveProcessorCount=2",
 		},
+		{
+			name:            "using experimentalContainerV2 sets Xms and Xmx and does not set ActiveProcessorCount",
+			launcherCustom:  "testdata/launcher-custom-experimental-container-v2.yml",
+			expectedJVMArgs: "-Xms3107979264 -Xmx3107979264",
+		},
+		{
+			name: "using experimentalContainerV2 with InitialRAMPercentage does not set Xms, Xmx, or " +
+				"ActiveProcessorCount",
+			launcherCustom:  "testdata/launcher-custom-experimental-container-v2-with-initial-ram-percentage.yml",
+			expectedJVMArgs: "-XX\\\\:InitialRAMPercentage=70.0",
+		},
+		{
+			name: "using experimentalContainerV2 with MaxRAMPercentage does not set Xms, Xmx, or " +
+				"ActiveProcessorCount",
+			launcherCustom:  "testdata/launcher-custom-experimental-container-v2-with-max-ram-percentage.yml",
+			expectedJVMArgs: "-XX\\\\:MaxRAMPercentage=70.0",
+		},
+		{
+			name:            "using experimentalContainerV2 does not use user-provided Xms or Xmx",
+			launcherCustom:  "testdata/launcher-custom-experimental-container-v2.yml",
+			expectedJVMArgs: "-Xms3107979264 -Xmx3107979264",
```

Comment:
What are your thoughts on just checking if the `Xms` and `Xmx` values are set in the test rather than a specific figure?